docs/src/running-validator/vote-accounts.md

```diff
@@ -0,0 +1,115 @@
+keypair file, or at a derived address from the authorized voter's public key.
+
+The address of a vote account does not need to have a stored corresponding
+private key, because the address (although it may be a valid public key) is never
+needed to sign any transactions, but is just used to look up the account
+information.
+
+When someone wants to [delegate tokens in a stake account](../staking.md),
+the delegation command is pointed at the vote account address of the validator
+to whom the token-holder wants to delegate.
+
+### Validator Identity
+
+The _validator identity_ is a system account that pays for all the vote transaction
+fees submitted by the vote account.  Because the vote account is expected to vote
+on most valid blocks it receives, the validator identity account is frequently
+(as often as once per slot, or roughly every 400ms) signing transactions and
+paying fees.  For this reason the validator identity account must be stored as
+a "hot wallet" as a keypair file on the same system the validator process is
+running.
+
+Because a hot wallet is generally less secure than an offline or "cold" wallet,
+we only recommend storing enough SOL on the Identity keypair to cover voting fees
+for some amount of time (perhaps a few weeks or months) and periodically topping
+off your validator identity account from a more secure wallet.
+
+The validator identity is required to be provided when a vote account is created.
+The validator identity can also be changed after an account is created by using
+the `vote-update-validator` command.
+
+### Vote Authority
+
+The _vote authority_ keypair is used to sign each vote transaction the validator
+node wants to submit to the cluster.  This can be the same keypair used as the
+validator identity for the same vote account, or a different keypair.  Because
+the vote authority, like the validator identity, is signing transactions
+frequently, this also must be a hot keypair on the same file system as the
+validator process.
+
+It is recommended to set the vote authority to the same address as the validator
+identity.  If the validator identity is also the vote authority, only one
+signature per vote transaction is needed in order to both sign the vote and pay
+the transaction fee.  Because transaction fees on Solana are assessed
+per-signature, having one signer instead of two will result in half the transaction
+fee paid compared to setting the vote authority and validator identity to two
+different accounts.
+
+The vote authority can be set when the vote account is created.  If it is not
+provided, the vote account with set the Validator Identity as the vote authority
+by default.  The vote authority can be changed later with the
+`vote-authorize-voter` command.
+
+The vote authority can be changed at most once per epoch.  If the authority is
+changed with `vote-authorize-voter`, this will not take effect until the end of
+the current epoch.  To support a smooth transition of the vote signing,
```

Comment:
```suggestion
changed with `vote-authorize-voter`, this will not take effect until the beginning
of the next epoch.  To support a smooth transition of the vote signing,
```